ReactNativeClient/lib/joplin-renderer/MdToHtml.js

```diff
@@ -17,6 +17,8 @@ const rules = {
 	code_inline: require('./MdToHtml/rules/code_inline'),
 	fountain: require('./MdToHtml/rules/fountain'),
 	mermaid: require('./MdToHtml/rules/mermaid').default,
+	sanitize_html: require('./MdToHtml/rules/sanitize_html').default,
+	video: require('./MdToHtml/rules/video'),
```

Comment:
Please move this to the `plugins` object below because your plugin is setup like a regular markdown-it plugin, unlike the rules in this object.

Reply:
UPDATE: I moved `video: require('./MdToHtml/rules/video')`, back to the rules object in joplin-renderer/MdToHtml.js, (I went through other me rules and every one in which context was added, was under `rules `object so I changed its import which was done in `plugin `object to `rules `object)